Alignment/APEEstimation/plugins/ApeEstimatorSummary.cc

```diff
@@ -812,11 +812,11 @@ ApeEstimatorSummary::calculateApe(){
        const double apeY2new(apeY2old + correctionY2);
        if(!smoothIteration || firstIter){
          apeX2 = apeX2new;
-	 apeY2 = apeY2new;
+        apeY2 = apeY2new;
        }
        else{
          apeX2 = std::pow(smoothFraction*std::sqrt(apeX2new) + (1-smoothFraction)*std::sqrt(apeX2old), 2);
-	 apeY2 = std::pow(smoothFraction*std::sqrt(apeY2new) + (1-smoothFraction)*std::sqrt(apeY2old), 2);
+        apeY2 = std::pow(smoothFraction*std::sqrt(apeY2new) + (1-smoothFraction)*std::sqrt(apeY2old), 2);
```

Comment:
@mteroerd the performance of `std::pow(A, 2)` is worse than `A*A`, i.e. I suggest to compute `A` and then use `A*A` instead of `std::pow`

Reply:
Done.